crates/dht/src/dht/tests.rs

```diff
@@ -316,7 +316,7 @@ async fn ring_sync_with_matching_disc() {
     // Put recent data in the first ring of both DHTs
     dht1.inject_ops(vec![MemoryOp::new(
         dht1.dht.partition.full_slice_end_timestamp(),
-        vec![7; 4],
+        vec![17; 4],
```

Comment:
Duplicate op id, the test was passing before the op store was more strict and it was valid that it was passing but broke with these changes. Better to just use unique op ids